packages/d3fc-webgl/examples/webgl-overlay.html

```diff
@@ -0,0 +1,13 @@
+<html>
```

Comment:
When running this example, if the browser window (and chart) is re-sized the WebGL and SVG  layers become mis-aligned. This might point to an underlying issue in d3fc-chart?

Reply:
Excellent spot, that wasn't fun to track down! 

I've added a new attribute to `d3fc-canvas` called `set-webgl-viewport` which propagates the surface dimensions down to the WebGL context. This is automatically set on the `webgl-plot-area` by `cartesianChart`. 

I didn't want to make this automatic in case there's a potential performance impact to requesting a WebGL context if you're not using it. As `webgl-plot-area` is only rendered if you supply a value to `webglPlotArea`, this shouldn't be a problem.

Reply:
great - no problem :-)